model/listeners.py

```diff
@@ -61,8 +64,11 @@ def site_configuration_has_changed(_db, cooldown=1):
 
 def _site_configuration_has_changed(_db, cooldown=1):
     """Actually changes the timestamp on the site configuration."""
-    now = datetime.datetime.utcnow()
+    now = utc_now()
     last_update = Configuration._site_configuration_last_update()
+    if last_update:
+        last_update = to_utc(last_update)
```

Comment:
It should be the responsibility of `._site_configuration_last_update()` to make sure the datetime it's returning is timezone-aware.